.github/workflows/check.yml

```
@@ -118,13 +115,12 @@ jobs:
         wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
         sudo apt-get update
         sudo apt-get -y install libpq-dev postgresql-${{ matrix.version }} postgresql-server-dev-${{ matrix.version }}
-        cargo install cargo-pgrx --git https://github.com/tensorchord/pgrx.git --rev $(cat Cargo.toml | grep "pgrx =" | awk -F'rev = "' '{print $2}' | cut -d'"' -f1) || true
+        cargo install cargo-pgrx --git https://github.com/tensorchord/pgrx.git --rev $(cat Cargo.toml | grep "pgrx =" | awk -F'rev = "' '{print $2}' | cut -d'"' -f1)
```

```suggestion
        (cargo install cargo-pgrx --git https://github.com/tensorchord/pgrx.git --rev $(cat Cargo.toml | grep "pgrx =" | awk -F'rev = "' '{print $2}' | cut -d'"' -f1)) || true
```

After researching, I found that shell script must add bracket in previous command like `(command) || true` to ignore the error code.

If we want to re-enable the cache for binary, we can add bracket for each commands.

---

Why would cargo return a status which is not 0?

---

Because when the cache was successfully restored, the `cargo-pgrx` had already been downloaded.

---

It seems that the broken action is caused by incorrect .so library version. If we re-enable cache for binaries, we may need to lock versions of system C/C++ dependencies. Do you have ideas about it?

---

Since that, disabling cache for binaries may be the better way.